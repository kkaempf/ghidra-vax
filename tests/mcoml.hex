d2 9f 30 00 14 20 9f 02 05 14 20
