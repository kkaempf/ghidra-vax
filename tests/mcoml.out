0x00000000: MCOML @#0x20140030, @#0x20140502
